br-webui/views/network.liquid

```diff
@@ -194,6 +194,37 @@ function setCompanionEthernetIP() {
 	}
 	alert("New IP will take affect after reboot");
 }
+function setCompanionDefault() {
+	document.getElementById("manual").checked=true;
+	setCompanionMode();
+}
+	
+function setCompanionMode() {
+	
+	var radios = document.getElementsByName("config");
+	if (radios[0].checked)
+	{
+		if (confirm("Are you sure you want to configure Companion as a DHCP server?")) {
+			socket.emit('set companion as dhcp server');
+			alert("For new changes to take effect, reboot the pi from the web interface and reconnect the ethernet cable to your computer after making necessary changes to your computer's network settings.");
+		}
+	}
+	else if (radios[1].checked)
+	{
+		if (confirm("Are you sure you want to configure Companion as DHCP client?")) {
+			socket.emit('set companion as dhcp client');
+			alert("Companion has been set to DHCP client. Find out the respective new IP address for the pi to reconnect.");
+		}
+	}
+	else if (radios[2].checked)
+	{
+		if (confirm("Are you sure you want to reset to defaults?")) {
+			socket.emit('set companion as manual');
+			alert("Companion has been set to manual mode. Please refer to Network Setup instructions to regain connection with your Pi.");
+		}
+	}		
+
+}
```

Comment:
This messages does not make clear for the user that companion will restart by itself.
Maybe something like:
```
alert("Are you sure ... ?\nCompanion will change the configurations and restart.")
```

Reply:
@mariyachanniwala please address this. I agree with @patrickelectric

Reply:
I already modified the messages after @patrickelectric reviewed it. I will modify it further.

Reply:
the problem is that the user is not aware that the system will reboot **until after they click the button**. The user needs to be made aware the system will reboot before they commit to that.